modules/apps/object/object-admin-rest-api/src/main/java/com/liferay/object/admin/rest/dto/v1_0/util/ObjectFieldUtil.java

```diff
@@ -43,8 +47,23 @@ public static JSONObject toJSONObject(ObjectField objectField) {
 		).put(
 			"label", objectField.getLabelMap()
 		).put(
-			"listTypeDefinitionId",
-			Long.valueOf(objectField.getListTypeDefinitionId())
+			"listTypeDefinitionExternalReferenceCode",
+			() -> {
+				if (StringUtil.equals(
+						objectField.getBusinessType(),
+						ObjectFieldConstants.BUSINESS_TYPE_PICKLIST)) {
+
+					ListTypeDefinition listTypeDefinition =
+						ListTypeDefinitionServiceUtil.getListTypeDefinition(
```

Comment:
@mateussgn @murilostodolni can we also change to use the `fetchListTypeDefinition` here?